staff/acct/sorry/sorry

```diff
@@ -74,12 +69,15 @@ def sorry_user(username, sorry_file):
 
     # WTF is final system check for?
     # log(rootstaffer, username)
-    send_log_mail(rootstaffer, username, open(reason_path, 'r').read())
+    with open(sorry_file, 'r') as filep:
+        send_log_mail(rootstaffer, username, filep.read())
 
     user_mail = get_user_real_mail(username)
     if user_mail:
-        send_sorry_mail(user_mail, username, open(reason_path, 'r').read())
-    kerberos_destroy()
+        with open(sorry_file, 'r') as filep:
+            send_sorry_mail(user_mail, username, filep.read())
```

Comment:
we may as well just put this whole block in the same `with` statement from directly above